TrashMob/client-app/src/components/Pages/MyDashboard.tsx

```diff
@@ -734,6 +706,36 @@ const MyDashboard: FC<MyDashboardProps> = (props) => {
         );
     }
 
+    /** Map hooks */
+    const upcomingEventsMap = useMap('upcoming-events-map')
+    const upcomingEvents = myEventList.filter(isUpcomingEvent)
+
+    // Zoom Map to show all Markers: upcomingEventsMap
+    useEffect(() => {
```

Comment:
These 2 useEffects are identical, why not create a hook like so:

```
const useZoomMapToFitMarkers = (map: google.maps.Map, events: EventData[]) => {
    useEffect(() => {
        if (map && events.length) {
            const bounds = new google.maps.LatLngBounds();
            events.forEach((e) => bounds.extend({ lat: e.latitude, lng: e.longitude }));
            map.fitBounds(bounds);
        }
    }, [map, events]);
};
```

Reply:
Good one, I'll do that